src/api/lib/authenticator.rb

```diff
@@ -147,14 +217,14 @@ def check_extracted_user
     unless @http_user
       if @login.blank?
         return true if check_for_anonymous_user
-        raise AuthenticationRequiredError.new
+        raise AuthenticationRequiredError
       end
-      raise AuthenticationRequiredError.new "Unknown user '#{@login}' or invalid password"
+      raise AuthenticationRequiredError, "Unknown user '#{@login}' or invalid password"
     end
 
     if @http_user.state == 'unconfirmed'
-      raise UnconfirmedUserError.new "User is registered but not yet approved. " +
-                                         "Your account is a registered account, but it is not yet approved for the OBS by admin."
+      raise UnconfirmedUserError, "User is registered but not yet approved. Your account " +
```

Comment:
@evanrolfe In my [tests](https://github.com/openSUSE/open-build-service/pull/2936/files#diff-cf7ae3ace32b891083995782628f1955R4), I didn't see any exceptions. Though in the test.log I was able to see those exceptions.

Could you check if that is a regression of the authenticator? Note that this is a [request test]( https://www.relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec). Though I briefly tested this with a controller test too.